flink/v1.13/flink/src/main/java/org/apache/iceberg/flink/FlinkCatalog.java

```diff
@@ -400,6 +400,24 @@ void createIcebergTable(ObjectPath tablePath, CatalogBaseTable table, boolean ig
     }
   }
 
+  // skip the AbstractConstraint comparison
```

Comment:
Could you write a note explaining why the Schema `equals` method doesn't work?

Reply:
that's right, I'll write the corresponding explanation